napari/utils/events/containers/_selection.py

```diff
@@ -90,7 +149,9 @@ def validate(cls, v, field: 'ModelField'):
 
         # no type parameter was provided, just return
         if not field.sub_fields:
-            return cls(data=data, current=current)
+            obj = cls(data=data)
+            obj.__current = current
```

Comment:
```suggestion
            obj._current_ = current
```

As above. Ignore all these if you don't like my suggestion, I don't love it either :joy:

Reply:
added